tests/unit/s2n_random_test.c

```diff
@@ -48,6 +48,8 @@
 #define NUMBER_OF_RANGE_FUNCTION_CALLS 200
 #define MAX_REPEATED_OUTPUT            4
 
+bool s2n_libcrypto_is_fips(void);
+bool s2n_libcrypto_is_openssl();
```

Comment:
Nit: looks a bit wonky to specify void for one no-arg method but not for the other

Reply:
we usually need `void` for compatibility with older compilers

Reply:
```suggestion
bool s2n_libcrypto_is_fips(void);
bool s2n_libcrypto_is_openssl(void);
```

Reply:
Fixed this and opened an issue for others in our codebase https://github.com/aws/s2n-tls/issues/4952